src/components/flowchart/index.js

```diff
@@ -213,7 +218,10 @@ export class FlowChart extends Component {
         const margin = 500;
         this.zoomBehaviour.translateExtent([
           [-sidebarWidth / scale - margin, -margin],
-          [width + margin + metaSidebarWidth / scale, height + margin]
+          [
+            width + margin + (metaSidebarWidth + codeSidebarWidth) / scale,
```

Comment:
With the introduction of the 2 new props to calculate the flowchart size, it is worth to add a test for flowchart to test the recalculation of the scale with different chartsize values?